docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -350,6 +350,23 @@ The following table includes some additional commands that might help you manage
 | Runs one iteration of `logrotate` manually, without changing the scheduled interval. | `sudo logrotate -f /etc/logrotate.conf --force` |
 | Reloads `syslog-ng`. | `sudo /usr/sbin/syslog-ng-ctl reload` |
 
+## UID2 Operator Error Codes
+
+The following table lists possible errors while setting up a Private Operator.
```

Comment:
The following table lists possible errors presented in the startup sequence of a Private Operator.

Reply:
I will also change error codes to be just E01, E02 etc and not AW, AZ, GC. Since it is generic errors pointing to provider specific web page. (Also it minimizes conditions in generic exception handler) 

See: https://github.com/IABTechLab/uid2-operator/pull/1232/files